api/src/modules/impact/dto/paginated-entities.dto.ts

```diff
@@ -1,6 +1,12 @@
+import { LOCATION_TYPES } from 'modules/sourcing-locations/sourcing-location.entity';
 import { ImpactTableEntityType } from 'types/impact-table-entity.type';
 import { PaginationMeta } from 'utils/app-base.service';
 export class PaginatedEntitiesDto {
-  entities: ImpactTableEntityType[];
+  entities: ImpactTableEntityType[] | LocationTypeEntities[];
```

Comment:
Can't the new grouping option be an optional value of 
`ImpactTableEntityType` ? Instead of making the entities prop be optional. 
This is normally a bad practice and it should be avoided when possible